spec/features/customers/show_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+describe 'Showing customer' do
+  before do
+    @customer = create :customer,
+                        address: "# Here the Customer address\n\nBla bla bla.",
+                        description: "Customer description"
+    login_as(create :admin)
+  end
+
+  it 'displays a customer' do
+    visit customer_path(@customer)
+
+    expect(page).to have_title 'Customer test customer - Project Manager'
+    expect(page).to have_active_navigation_items 'Customer', 'List of Customer'
+    expect(page).to have_breadcrumbs 'Project Manager', 'Customer', 'Customer test customer'
+    expect(page).to have_headline 'Customer test customer'
+
+    within dom_id_selector(@customer) do
+      expect(page).to have_css '.created_at', text: '15 Jun 14:33'
+      expect(page).to have_css '.updated_at', text: '15 Jun 14:33'
+
+      within '.address' do
+        expect(page).to have_css 'h2', text: 'Address'
+        expect(page).to have_content,  text: '# Here the Customer address Bla bla bla.'
```

Comment:
Du verwendest `have_content` wie `have_css`, aber die verhalten sich nicht gleich!

Korrekt:

    expect(page).to have_content '# Here the Customer address Bla bla bla.'